src/shared/components/query/GeneSelector.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
```

Comment:
This file is renamed yes?

Reply:
Yes. It was the previous GeneSetSelector. Just to avoid confusion. I did the same for the GeneSetValidator and GeneValidator files.

Reply:
You can test the PR here: http://cbioportal-test.thehyve.net/cbioportal/